test/common/http/http1/codec_impl_test.cc

```diff
@@ -2635,7 +2713,8 @@ TEST_F(Http1ClientConnectionImplTest, LowWatermarkDuringClose) {
 
   EXPECT_CALL(response_decoder, decodeHeaders_(_, true))
       .WillOnce(Invoke([&](ResponseHeaderMapPtr&, bool) {
-        // Fake a call for going below the low watermark. Make sure no stream callbacks get called.
+        // Fake a call for going below the low watermark. Make sure no stream callbacks get
+        // called.
```

Comment:
Unclear why the previous 3 whitespace changes were done.  Was it the fix format script or a consequence of some indent change that was later backed out?

Reply:
The latter, I've reverted it.